### ISSUE TITLE
Are PhaseNet weights in SeisBench transferred or retrained? (PyTorch vs. TF)

### ISSUE THREAD

**Issue by XDevKoi** [seisbench/seisbench#412]
Hello! I have a question about the PhaseNet weights provided in SeisBench.

The original TensorFlow version by Dr. Zhu utilizes an augmentation method called stack_event (helpful for multi-event learning in a single window). However, this method doesn't seem to be included in seisbench.generate.

<img width="952" height="609" alt="Image" src="https://github.com/user-attachments/assets/c3e2eadf-2254-4ae6-852d-1b4f6c88cc6f" />

Given this difference in augmentation, I'm curious about the PyTorch weights (it would be highly appreciated if some comparison metrics could be provided):

    - Are the current PhaseNet weights in SeisBench directly transferred from the original TensorFlow model?

    - Or was the model retrained from scratch using the SeisBench pipeline?

Thanks for your help and for maintaining this great repository!

**Comment by yetinam** [comment]
Hi @XDevKoi ,
it's a mix. The "original" weights are converted from TF, so they are exactly the weights published by Zhu & Beroza. All other weights have been retrained. Most of them with the workflow in [this repository](https://github.com/seisbench/pick-benchmark), but some have also been trained later by other teams with different workflows. You can find details in the `weights_docstring` argument, after loading models with `from_pretrained`.
As for the augmentation, it was indeed missing in our training, so only the `original` weights have used this augmentation.
In terms of comparison metric, the best that we've created is [this paper](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2021JB023499), but it doesn't include all the weights available in SeisBench now.
Let me know if you have any further questions!
